### DISCUSSION TITLE
Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by HamzaAryanSapnil** [community/community#121507] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Though i have set up everything successfully , I got this error and copilot is not working
Invalid copilot token: missing token: 403

**Comment by dogukan33** [comment]
Hello, 

I previously had that issue and fixed with the helps of the discussion below. Let me know if it not works for you.

[this one](https://github.com/orgs/community/discussions/47319#discussioncomment-4967885)

**Comment by HamzaAryanSapnil** [comment]
hello,
how can i get this access token

**Comment by davecheney** [comment, marked as answer]
@HamzaAryanSapnil your copilot access has been suspended due to a failure to process payment. 
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
